test/e2e/lib/components/secure-payment-component.js

```diff
@@ -26,7 +27,8 @@ export default class SecurePaymentComponent extends AsyncBaseContainer {
 		);
 		this.personalPlanSlug = getJetpackHost() === 'WPCOM' ? 'personal-bundle' : 'jetpack_personal';
 		this.premiumPlanSlug = getJetpackHost() === 'WPCOM' ? 'value_bundle' : 'jetpack_premium';
-		this.businessPlanSlug = getJetpackHost() === 'WPCOM' ? 'business-bundle' : 'jetpack_business';
+		this.businessPlanSlug =
+			getJetpackHost() === 'WPCOM' ? 'business-bundle' : 'jetpack_security_daily';
```

Comment:
Jetpack Professional (business) is no longer a thing so I replaced its slug with one of an actual Jetpack plan.